### PR TITLE
[doc] Add redis-cluster help document

### PR DESCRIPTION
**Pull request by zzzk1**
## What's changed?
add redis cluster help document.

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [x]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

**Comment by zzzk1** [comment]
@tomsun28 pls take a look~

File: home/docs/help/redis_cluster.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+   ```
+4. Specific operations.
+
+   Add a redis monitor center, fill require parameters.
+
+   ![](img/docs/help/redis-cluster-add.png)
+
+   final effect.
+
+   ![](img/docs/help/redis-cluster-view.png)
```
Comment (LiuTianyou) [inline home/docs/help/redis_cluster.md:153]: The image path is incorrect and needs to be changed to `/img/docs/help/xxx`

File: home/docs/help/redis_cluster.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+### Pre-monitoring operations
```
Comment (LiuTianyou) [inline home/docs/help/redis_cluster.md:1]: Here need to add a file header in first line ,like this：
```
---
id: redis_cluster
title: Monitoring Redis Cluster
sidebar_label: Redis Cluster Monitor
keywords: [ open source monitoring tool, open source Redis Cluster monitoring tool, monitoring Redis Cluster metrics ]
---

```

File: home/i18n/en/docusaurus-plugin-content-docs/version-v1.1.x/help/redis_cluster.md
Comment (zuobiao-zhou) [inline home/i18n/en/docusaurus-plugin-content-docs/version-v1.1.x/help/redis_cluster.md:1]: Why add this doc to "version-v1.1.x" directory?

**Comment by zhangshenghang** [comment]
This PR has not been updated for almost a month,  I will continue to complete it

**Pull request closed without merging**